src/Psy/Shell.php

```diff
@@ -302,6 +303,7 @@ public function doRun(InputInterface $input, OutputInterface $output)
         // }
 
         $this->output->writeln($this->getHeader());
+        $this->output->writeln($this->getVersionCheckResult());
```

Comment:
Does this add an extra newline when there's nothing to report from the version checker?